File: DesktopSample/DataBinding/PageAndRDL/OleDbDataSource/C#/OleDbDataSource/MainForm.cs
using System;
using System.Windows.Forms;
using System.IO;
using GrapeCity.ActiveReports;

namespace ActiveReports.Samples.OleDbDataSource
{
	partial class MainForm : Form
	{
		public MainForm()
		{
			InitializeComponent();
		}

		// レポートをロードして表示します。
		private void MainForm_Load(object sender, EventArgs e)
		{
			var rptPath = new FileInfo(@"..\..\..\OleDBReport.rdlx");
			var pageReport = new PageReport(rptPath);
			reportPreview.LoadDocument(pageReport.Document);
		}
	}
}

File: DesktopSample/DataBinding/PageAndRDL/OleDbDataSource/VB.NET/OleDbDataSource/MainForm.vb
Imports System.IO
Imports System.Text
Imports GrapeCity.ActiveReports

Public Class MainForm

	Sub New()
		Encoding.RegisterProvider(CodePagesEncodingProvider.Instance)
		InitializeComponent()
	End Sub

	' レポートをロードして表示します。
	Private Sub MainForm_Load(ByVal sender As System.Object, ByVal e As EventArgs) Handles MyBase.Load
		Dim rptPath As New FileInfo("..\..\..\OleDBReport.rdlx")
		Dim pageReport As New PageReport(rptPath)
		reportPreview.LoadDocument(pageReport.Document)
	End Sub

End Class
